src/nox_uv/__init__.py

```diff
@@ -54,31 +59,39 @@ def session(
             uv_all_extras=uv_all_extras,
             uv_all_groups=uv_all_groups,
             uv_only_groups=uv_only_groups,
+            uv_sync_locked=uv_sync_locked,
             **kwargs,
         )  # type: ignore
 
     [function] = args
 
     # Create the `uv sync` command
-    sync_cmd = ["uv", "sync", "--no-default-groups", "--locked"]
+    sync_cmd = ["uv", "sync", "--no-default-groups"]
+    extended_cmd: list[str] = []
```

Comment:
Why do we need two variables? Can't we simply append `--locked` like we do all of the other commands to a single variable?